src/main/java/org/broadinstitute/consent/http/models/UserFile.java

```diff
@@ -0,0 +1,136 @@
+package org.broadinstitute.consent.http.models;
+
+import org.broadinstitute.consent.http.enumeration.UserFileCategory;
+
+import java.io.InputStream;
+import java.util.Date;
+import java.util.Objects;
+
+public class UserFile {
+
+    public static final String USER_FILE_QUERY_WITH_UF_PREFIX =
+            "uf.user_file_id as uf_use_file_id, "
+            + " uf.entity_id as uf_entity_id, "
+            + " uf.file_name as uf_file_name, "
+            + " uf.bucket_name as uf_file_name, "
+            + " uf.blob_id as uf_file_name, "
+            + " uf.category as uf_file_name, "
+            + " uf.media_type as uf_file_name, "
+            + " uf.create_user_id as uf_file_name, "
+            + " uf.create_date as uf_create_date, "
+            + " uf.deleted as uf_deleted ";
+
+    private Integer userFileId;
+    private String entityId;
+    private String fileName;
+    private String bucketName;
```

Comment:
Would it be better to type this as a UUID?

Reply:
Do we even need a bucket name? The general bucket we use is an environment-specific location, so we have separate ones for dev, staging, prod, etc. A `BlobId` should be enough.

Reply:
wasn't sure if it would be possible for the bucket to change in the future, so I was tracking it just to be safe

Reply:
Ok, now that I'm tracking, could we refer to blob_id as blobId?